src/utils/__tests__/model-name-convert.test.mjs

```
@@ -0,0 +1,296 @@
+      const originalClaude = actualMockModels['claude-2'];
+      actualMockModels['claude'] = { desc: "Claude Base", value: "claude"};
+      expect(modelNameToDesc('claude-200k', t)).toBe('Claude Base (200k)'); // presetPart='claude', customPart='200k'
+
+      // Test with a preset part that is in Models and custom part in ModelMode
+      // modelNameToPresetPart('gpt-4-fast') -> 'gpt'. Let's assume 'gpt' is not in Models.
+      // This test needs a presetPart that IS in Models.
+      // Let's use a hypothetical 'claude-fast'. modelNameToPresetPart('claude-fast') -> 'claude'
+      expect(modelNameToDesc('claude-fast', t)).toBe('Claude Base (Fast Mode)');
+
+      // Restore actualMockModels if changed
+      delete actualMockModels['claude'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)